<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>331</width>
     <height>328</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="userNameL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userNameF">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #2471FF;
    border-radius: 10px;
    padding: 6px;
    font-size: 14px;
    color: #000000;
}

QLineEdit:focus {
    border: 2px solid #164499;
    background-color: #F5F8FF;
}
</string>
      </property>
      <property name="placeholderText">
       <string>Enter User Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="EmailL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailF">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #2471FF;
    border-radius: 10px;
    padding: 6px;
    font-size: 14px;
    color: #000000;
}

QLineEdit:focus {
    border: 2px solid #164499;
    background-color: #F5F8FF;
}
</string>
      </property>
      <property name="placeholderText">
       <string>Enter Your Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="passwordL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordF">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #2471FF;
    border-radius: 10px;
    padding: 6px;
    font-size: 14px;
    color: #000000;
}

QLineEdit:focus {
    border: 2px solid #164499;
    background-color: #F5F8FF;
}
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Enter Your Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ConfirmPasswordL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm Password</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ConfirmpasswordF">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 2px solid #2471FF;
    border-radius: 10px;
    padding: 6px;
    font-size: 14px;
    color: #000000;
}

QLineEdit:focus {
    border: 2px solid #164499;
    background-color: #F5F8FF;
}
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2471FF;
    color: white;
    border-radius: 10px;
    font-size: 15px;
    font-weight: bold;
    padding: 8px 15px;
}

QPushButton:hover {
    background-color: #164499;
}

QPushButton:pressed {
    background-color: #0E326F;
}
</string>
      </property>
      <property name="text">
       <string>SignUp</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="appBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FF2471FF;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">Color:#FFFFFF</string>
    </property>
    <property name="text">
     <string>SignUp</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="cornerDecor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#cornerDecor {
    background-color: #2471FF;
    border-top-right-radius: 100px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF; background-color: #FF2471FF;</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="alreadyHA">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Already hava an account? </string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="goToLogin">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #2471FF;
    font-weight: bold;
}
QLabel:hover {
    text-decoration: underline;
    cursor: pointer;
}
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
